src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToCopyLocalDependencies.cs

```diff
@@ -146,7 +145,203 @@ public void It_copies_local_specific_runtime_package_dependencies_on_build()
+        [Fact]
+        public void It_does_not_copy_local_package_dependencies_for_netstandard_projects()
+        {
+            const string ProjectName = "TestProjWithPackageDependencies";
+
+            TestProject testProject = new TestProject()
+            {
+                Name = ProjectName,
+                IsSdkProject = true,
+                TargetFrameworks = "netstandard2.0"
+            };
+
+            testProject.PackageReferences.Add(new TestPackageReference("Newtonsoft.Json", "11.0.2"));
+            testProject.PackageReferences.Add(new TestPackageReference("sqlite", "3.13.0"));
+
+             var testProjectInstance = _testAssetsManager
+                .CreateTestProject(testProject)
+                .Restore(Log, ProjectName);
+
+            var buildCommand = new BuildCommand(Log, testProjectInstance.TestRoot, ProjectName);
+
+            buildCommand.Execute()
+                .Should()
+                .Pass();
+
+            var outputDirectory = buildCommand.GetOutputDirectory(testProject.TargetFrameworks);
+            outputDirectory.Should().OnlyHaveFiles(new[] {
+                $"{ProjectName}.deps.json",
+                $"{ProjectName}.dll",
+                $"{ProjectName}.pdb"
+            });
+        }
+
+        [Fact]
+        public void It_copies_local_package_dependencies_for_netstandard_projects_when_requested_to()
+        {
+            const string ProjectName = "TestProjWithPackageDependencies";
+
+            TestProject testProject = new TestProject()
+            {
+                Name = ProjectName,
+                IsSdkProject = true,
+                TargetFrameworks = "netstandard2.0"
+            };
+
+            testProject.AdditionalProperties["CopyLocalLockFileAssemblies"] = "true";
+            testProject.PackageReferences.Add(new TestPackageReference("Newtonsoft.Json", "11.0.2"));
+            testProject.PackageReferences.Add(new TestPackageReference("sqlite", "3.13.0"));
+
+             var testProjectInstance = _testAssetsManager
+                .CreateTestProject(testProject)
+                .Restore(Log, ProjectName);
+
+            var buildCommand = new BuildCommand(Log, testProjectInstance.TestRoot, ProjectName);
+
+            buildCommand.Execute()
+                .Should()
+                .Pass();
+
+            var outputDirectory = buildCommand.GetOutputDirectory(testProject.TargetFrameworks);
+            outputDirectory.Should().OnlyHaveFiles(new[] {
+                $"{ProjectName}.deps.json",
+                $"{ProjectName}.dll",
+                $"{ProjectName}.pdb",
+                "Newtonsoft.Json.dll",
+                "runtimes/linux-x64/native/libsqlite3.so",
+                "runtimes/osx-x64/native/libsqlite3.dylib",
+                "runtimes/win7-x64/native/sqlite3.dll",
+                "runtimes/win7-x86/native/sqlite3.dll"
+            });
+        }
+
+        [Fact]
+        public void It_copies_local_all_assets_on_self_contained_build()
```

Comment:
I'd recommend finding the existing self-contained publish tests, and updating them to also cover self-contained build (ie via a Theory).

Reply:
Do we want to do that in addition to this test?  I feel like we should have at least one test covering self-contained builds from the "build" tests.

Reply:
Yes, this test is fine, I just think that there are a bunch more self-contained scenarios that we already have test coverage for, so we should apply them to self-contained build.